app/src/main/java/com/example/dispatcher/domain/homepage/repo/InterfaceArticleRepo.kt

```diff
@@ -0,0 +1,7 @@
+package com.example.dispatcher.domain.homepage.repo
+
+import com.example.dispatcher.presentation.homepage.model.Article
+
+interface InterfaceArticleRepo {
```

Comment:
not good naming, should be something like: ArticleRepository, IArticleRepository